pkg/mountutil/mountutil.go

```diff
@@ -29,63 +30,83 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-func ParseFlagV(s string, volStore volumestore.VolumeStore) (*specs.Mount, error) {
+type Processed struct {
+	Mount           specs.Mount
+	AnonymousVolume string // name
+}
+
+func ProcessFlagV(s string, volStore volumestore.VolumeStore) (*Processed, error) {
+	var (
+		res      Processed
+		src, dst string
+		ro       bool
+	)
 	split := strings.Split(s, ":")
-	if len(split) < 2 || len(split) > 3 {
-		return nil, errors.Errorf("failed to parse %q", s)
-	}
-	src, dst := split[0], split[1]
-	if !strings.Contains(src, "/") {
-		// assume src is a volume name
-		vol, err := volStore.Get(src)
+	switch len(split) {
+	case 1:
+		dst = s
```

Comment:
Don't we need a check if `dst` is an absolute path?

Reply:
Surprisingly, we don't need: `docker run -v foo alpine` mounts an anon volume on `/foo`